### PR TITLE
updated sample queries doc

### PR DESCRIPTION
**Pull request by bartr**
# Purpose of PR

- updated sample queries doc

## Type of PR

- [x] Documentation changes
- [ ] Labs changes
- [ ] Codespaces changes
- [ ] Code changes
- [ ] Test changes

### CODE REVIEW

File: labs/res-edge-labs/deploy-res-edge/sample-queries.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+> The CLI commands are in `preview` and will likely change before final release
+
+- The `kic CLI` implements basic commands to query the `Data Service`
+- Additional CLI commands and a web user interface are planned for future releases
+
+## Data Service Objects
+
+The `Data Service` exposes the following entities via a REST/OData interface
+
+- Application - represents an application `registered` by an app dev team
```
Comment (madossan01) [inline labs/res-edge-labs/deploy-res-edge/sample-queries.md:12]:
```suggestion
- Application - represents an application `registered` by an Application Dev team
```

**Review by mahomedalid — APPROVED** [review]
LGTM

File: labs/res-edge-labs/deploy-res-edge/sample-queries.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
+# Sample Data Service Queries
+
+> The CLI commands are in `preview` and will likely change before final release
+
+- The `kic CLI` implements basic commands to query the `Data Service`
+- Additional CLI commands and a web user interface are planned for future releases
+
+## Data Service Objects
```
Comment (madossan01) [inline labs/res-edge-labs/deploy-res-edge/sample-queries.md:8]:
```suggestion
## Data Service Entities
```

Comment (bartr) [reply labs/res-edge-labs/deploy-res-edge/sample-queries.md:8]: Data Services return REST objects, so objects feels correct

File: labs/res-edge-labs/deploy-res-edge/sample-queries.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+The `Data Service` exposes the following entities via a REST/OData interface
+
+- Application - represents an application `registered` by an app dev team
+  - An Application belongs to zero or one Namespace
+- Cluster - represents a K8s physical cluster
+  - Namespaces are assigned to a Cluster directly or via Group membership
```
[the remark this answers is not in the document]
Comment (bartr) [reply labs/res-edge-labs/deploy-res-edge/sample-queries.md:15]: alpha gives us consistency

Each object has dependencies on other objects. I'm not aware of an order that allows us to go in dependency order.

If there is an order that is better, we can change all of the references.

File: labs/res-edge-labs/deploy-res-edge/sample-queries.md (hunk 1 above)
Comment (bartr) [reply labs/res-edge-labs/deploy-res-edge/sample-queries.md:12]: fixed

**Pull request merged**